### PR TITLE
Migrate the unary OP evaluate interface test into the unary_ops.cpp

### PR DESCRIPTION
**Pull request by luweizhou2016**
### Details:
 - *Generlize NGraph unary op_eval test cases to avoid duplicated code*
 - *Current existing Mish, SoftPlus, HSigmoid, HSwish are migrated into*

### Tickets:
 - *58644*

### CODE REVIEW

**Comment by jdanieck** [comment]
@ilyachur @tomdol I can't find any good reason to have these `op_eval` tests, looks like it's a legacy approach - we could do the same with backend tests. I propose to drop these type of tests entirely and stick to `backend` only - it will be more clear if we have less types of tests. What do you think about it ? If we do so, then this PR won't be needed at all.

File: ngraph/test/op_eval/unary_ops.cpp [hunk 1]
```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
```
Comment (ilyachur) [inline ngraph/test/op_eval/unary_ops.cpp:5]: I don't want to merge several operations to one file. Can we use the same approach with parameterized tests as we use in visitor tests? Each operation has separate test file.

Moreover I dislike op_eval tests which test checks that evaluate method works as expected. But we moved INTERPRETER backend to evaluate methods, it means that we can add only backend tests instead op op_eval.

At the current moment I propose to add only backend tests for new operations and do not add new op_eval tests.

**Comment by ilyachur** [comment]
> @ilyachur @tomdol I can't find any good reason to have these `op_eval` tests, looks like it's a legacy approach - we could do the same with backend tests. I propose to drop these type of tests entirely and stick to `backend` only - it will be more clear if we have less types of tests. What do you think about it ? If we do so, then this PR won't be needed at all.

@jdanieck  Please see my comment to this PR 😄 I have the same opinion as you.

**Comment by jdanieck** [comment]
> > @ilyachur @tomdol I can't find any good reason to have these `op_eval` tests, looks like it's a legacy approach - we could do the same with backend tests. I propose to drop these type of tests entirely and stick to `backend` only - it will be more clear if we have less types of tests. What do you think about it ? If we do so, then this PR won't be needed at all.
> 
> @jdanieck Please see my comment to this PR 😄 I have the same opinion as you.

@ggalieroc could you create a ticket to migrate these `op_eval` tests to backend? It's a nice ramp-up task for new intern :)

**Comment by luweizhou2016** [comment]
> > > @ilyachur @tomdol I can't find any good reason to have these `op_eval` tests, looks like it's a legacy approach - we could do the same with backend tests. I propose to drop these type of tests entirely and stick to `backend` only - it will be more clear if we have less types of tests. What do you think about it ? If we do so, then this PR won't be needed at all.
> > 
> > 
> > @jdanieck Please see my comment to this PR smile I have the same opinion as you.
> 
> @ggalieroc could you create a ticket to migrate these `op_eval` tests to backend? It's a nice ramp-up task for new intern :)

Great. I was also confused about OP->evaluate() interface(if having) should been validated by  int_backend call() method. I will drop the PR and close the JIRA later. Thanks!

**Pull request closed without merging**